prow/jobs/incubator/reconciler/reconciler.yaml

```diff
@@ -275,6 +275,79 @@ periodics: # runs on schedule
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: periodic-main-kyma-incubator-reconciler-auto-imagebump
```

Comment:
We only use use `pre` and `post` prefixes for pre-submit and post-submit jobs. If there is no prefix, that means the job is periodic, so you don't need to use periodic in the name. You can use a job name like `reconciler-control-plane-image-bump` to make it simpler and easier to understand what it is about.

Reply:
Thats a good piece of information. Thanks! I will update it.

Reply:
updated